lab_1_classify_by_unigrams/main.py

```diff
@@ -2,7 +2,6 @@
 Lab 1
 Language detection
 """
-import json
 
 
 def tokenize(text: str) -> list[str] | None:
```

Comment:
Is it your implementation of this function?

Reply:
I used to have another implementation (I guess it means the way in which code is realized) of tokenize() which I then simplified using list comprehension.